_posts/2023-11-01-sr2024-kickstart-and-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SR2024 Kickstart Out Of This World
+---
+
+Student Robotics 2024 was Kickstarted on Saturday 21st October on our livestream. 30 teams from around the UK and Europe joined us as we introduced competitors to the competition, our Kits, and this year’s game! Teams will have six months to build, program, and test a fully autonomous robot, learning invaluable technical and soft skills along the way.
+
+This year's game, Final Frontier, plunges teams into an interstellar battle for dominance among four formidable civilizations, all vying for control of the cosmos. Teams must retrieve asteroids scattered throughout the asteroid field and transport them back to their home planet or, even better, their intergalactic spaceship currently under construction. However, the stakes are high, as within the confines of the asteroid belt lies The Egg, a cosmic time bomb that, if disturbed, could spell chaos. Teams must strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With asteroids worth points, spaceships to manage, and The Egg lurking, the universe has never been this competitive. May the most astute civilization prevail in this high-stakes cosmic showdown! For complete details, including prizes up for grabs this year, you can consult our [rulebook][rules].
+
+Following on from last year’s success, we’re continuing to offer teams a set of Challenges to be completed throughout the competition year. These encourage teams to start work on their robots early and cover movement, sensors, and vision. They may be approached in any order, and completing challenges before certain deadlines will earn the teams invaluable bonus league points. Full details are available in the rulebook and the deadlines are on our [events page][events].
+

```

Comment:
Maybe add here something along the lines of the following?

> Over the next six months the teams will work out for themselves how best to approach the design & build of their robot. This freedom of approach generates a wide variety of robot entrants to the final competition, while teaching the competitors valuable skills along the way. Teams will learn an assortment of technical skills from programming for real interactions with the world to building the hardware that makes up their robot. They will also gain practical experience working as a team on a large unstructured engineering task.

(maybe s/unstructured/open-ended/ ?)

Reply:
👍 Thanks for this great inspiration! Tweaked into [7b86937](https://github.com/srobo/website/pull/533/commits/7b86937f96557ab8f5a5abff3f70b654c2295773)